nni/retiarii/strategy/bruteforce.py

```diff
@@ -121,4 +121,7 @@ def run(self, base_model, applied_mutators):
                     if budget_exhausted():
                         return
                     time.sleep(self._polling_interval)
-                submit_models(get_targeted_model(base_model, applied_mutators, sample))
+                try:
+                    submit_models(get_targeted_model(base_model, applied_mutators, sample))
+                except InvalidMutation as e:
+                    _logger.warning(f'Invalid mutation: {e}. Skip.')
```

Comment:
you only updated bruteforce strategies, what about other strategies?

Reply:
It's not easy to insert retry logic into algorithms like RL or EVO, because they might hang. I leave the algorithm work in future PRs.